### PR TITLE
TPD-251 Adds FinanceAdmin role to role requirements

### PR DESCRIPTION
**Pull request by asherlloyd**
FinanceAdmin role was missing from the list of required roles.

### CODE REVIEW

**Comment by brondavies** [comment]
You'll have to add the `FinanceAdmin` role to `Areas/Manage/Controllers/TransactionsController.cs` as well.

**Comment by asherlloyd** [comment]
> You'll have to add the `FinanceAdmin` role to `Areas/Manage/Controllers/TransactionsController.cs` as well.

Did you mean BatchController.cs?

**Comment by brondavies** [comment]
> @brondavies Did you mean BatchController.cs?

Yeah, probably 😄

File: CmsWeb/Areas/Manage/Views/Transactions/Index.cshtml [hunk 1]
```diff
@@ -45,51 +45,89 @@
...
+                            </label>
+                        </div>
+                    </div>
+                </div>
+            }
+            <div class="col-lg-3 col-md-3 col-sm-6">
+                <div class="form-group">
+                    @if (ViewBag.db.Setting("ShowSyncBatchDatesOption"))
+                    {
+                        <a tabindex="0" href="#" data-toggle="popover" data-placement="bottom" data-trigger="focus" data-title="Additional Filters"><i class="fa fa-info-circle"></i></a>&nbsp;
```
Comment (brondavies) [inline CmsWeb/Areas/Manage/Views/Transactions/Index.cshtml]: @asherlloyd the compiler is complaining about the `&nbsp;` on this line.

File: CmsData/Migrations/20200214_ShowSyncBatchDatesOption_setting.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿IF NOT EXISTS(select 1 from dbo.SettingMetadata where SettingId = 'ShowSyncBatchDatesOption')
+BEGIN
+    DECLARE @contributionsCategoryId INT = 3, @dataBool INT = 1;
+
+    SET IDENTITY_INSERT dbo.SettingCategory ON;
```
Comment (brondavies) [inline CmsData/Migrations/20200214_ShowSyncBatchDatesOption_setting.sql]: This doesn't need to be set and it doesn't reset it later

**Pull request merged**